docs/astro/files/operators/get_file_list.rst

```diff
@@ -6,7 +6,11 @@ get_file_list
 
 When to use the ``get_file_list`` operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We can use ``get_file_list`` when you want a list file available in storage based on storage path pattern and Airflow connection. This might useful when you want to create dynamic task on the response of this operator.
+You can use ``get_file_list`` retrieve a list of available files based on a storage path and an Airflow connection. This can be useful for dynamically generating tasks based on which files are available on your storage system.
```

Comment:
```suggestion
You can use ``get_file_list``to retrieve a list of available files based on a storage path and the Airflow connection. Based on the files available on your system storage, this can generate tasks dynamically.
```